src/ElectricPokemon.java

```diff
@@ -0,0 +1,41 @@
+public class ElectricPokemon extends Pokemon{
+    final private String type;
```

Comment:
Je hebt in elke pokemon klasse de final private String type; gezet, hoe zou je dit makkelijker kunnen maken dat een pokemon altijd een type heeft?